src/main/kotlin/com/example/ajaxproject/service/GroupChatServiceImpl.kt

```diff
@@ -120,4 +119,12 @@ class GroupChatServiceImpl(
             throw WrongActionException("Admin cant leave a chat")
         }
     }
+    fun toResponseDto(chatMessage: GroupChatMessage): GroupChatMessageResponse {
```

Comment:
```suggestion

    fun toResponseDto(chatMessage: GroupChatMessage): GroupChatMessageResponse {
```

Reply:
why this discussion was silently resolved?

You have not applied the suggested changes